Initial styling for form

helloimela opened the pull request:
- Added form styling
- Added variable for border color in table and form

**Solving issue**</br>
Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)
Fixes: #15

Commit:
Initial styling for form

- Added form styling
- Added variable for border color in table and form

mmexvr commented on src/styles/_variables.scss:
```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
+$input-padding-y:             1rem;
+$input-padding-y-sm:          0.7rem;
+$input-padding-x-sm:          1.2rem;
+$input-border-radius:         3px;
```
Do we need a specific border-radius for input, can't we use the border-radius variable?

helloimela replied:
I will add global border-radius after I merge master to this branch

mmexvr commented on src/styles/components/_forms.scss:
```diff
@@ -0,0 +1,87 @@
+  }
+  
+  // &:-ms-input-placeholder { /* Internet Explorer 10-11 & Edge */
+  //   color: $dark-01;
```
Can probebly remove this for now, since we don't support anything with IE10 and we have the same rule above

helloimela replied:
Okay, tested this, unfortunately need to keep it separated to make it work :(

mmexvr commented on src/styles/_variables.scss:
```diff
@@ -110,9 +114,19 @@ $h4-font-size-article:        $font-size-article * 1.33;
+$table-border-color:          $border-color;
+$input-border-color:          $border-color;
+
+$input-padding-x:             1.2rem;
```
Think this is more in taste, but can't we have one rule for padding? instead of two for sizing?

helloimela replied:
So, to keep the height of the input according to the visual guidelines we need to do either one of these:
1. Add different padding
Or
2. Set fix height on the input (a solution we had in CUI 3).
I prefer solution 1 though, feel like we should avoid fixed height on element.

mmexvr commented:
Conflict

Commit:
Merge branch 'master' into improvement/form_styling

Commit:
Add improvement for the review

- Added placeholder color for different browser
- Added variable value from global value

Commit:
Merge branch 'master' into improvement/form_styling
